### PR TITLE
Check for null TtlAction in DefaultFileSystemMaster

### PR DESCRIPTION
**Pull request by yuzhu**
cherry-pick of PR Alluxio/alluxio#8511
cherry-pick of commit Alluxio/alluxio@b6ede350964986fb51b26cf0304847bb53ebb3a3

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
```
Failed tests: 
  FileSystemIntegrationTest.testTtlActionSetAttribute:345 TTL action should be same expected:<FREE> but was:<DELETE>
```

**Comment by yuzhu** [comment]
@gpang test fixed, PTAL. thanks

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -3624,7 +3624,9 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
     TtlAction action = options.getTtlAction();
 
     if (ttl != null && inode.getTtl() != ttl) {
+      mTtlBuckets.remove(inode);
       inode.setTtl(ttl);
+      mTtlBuckets.insert(inode);
```
Comment (ZacBlanco) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: This touches the DefaultFileSystemMaster and not a test. Is this something that should also go into master?

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**